ui/src/app/views/data-page/component.tsx

```diff
@@ -0,0 +1,253 @@
+            navigate(['workspaces', namespace, id,  'cohorts', 'build']);
+          }}>
+            <div style={styles.cardHeader}>
+              <h2 style={styles.cardHeaderText}>Cohorts</h2>
+              <ClrIcon shape='plus-circle' class='is-solid' size={18} style={{marginTop: 5}}/>
+            </div>
+            <div style={styles.cardText}>
+              Littera gothica quam nunc putamus parum claram anteposuerit litterarum
+              formas humanitatis per. In iis qui facit eorum claritatem Investigationes
+              demonstraverunt lectores legere me lius quod ii legunt.
+            </div>
+          </CardButton>
+          <CardButton style={{...styles.resourceTypeButton, ...styles.resourceTypeButtonLast}}
+                      onClick={() => {
+                        this.setState({creatingConceptSet: true});
+                      }}>
+            <div style={styles.cardHeader}>
+              <h2 style={styles.cardHeaderText}>Concept Sets</h2>
+              <ClrIcon shape='plus-circle' class='is-solid' size={18} style={{marginTop: 5}}/>
+            </div>
+            <div style={styles.cardText}>
+              Littera gothica quam nunc putamus parum claram anteposuerit litterarum
+              formas humanitatis per. In iis qui facit eorum claritatem Investigationes
+              demonstraverunt lectores legere me lius quod ii legunt.
+            </div>
+          </CardButton>
+        </div>
+      </FadeBox>
+      <FadeBox style={{marginTop: '1rem'}}>
+        <div style={styles.tabContainer}>
+          <h2 style={{margin: 0, color: '#2F2E7E', fontSize: '16px', fontWeight: 600}}>Show:</h2>
+          <TabButton active={activeTab === Tabs.SHOWALL} onClick={() => {
+            this.setState({
+              activeTab: Tabs.SHOWALL
+            });
+          }}>Show All</TabButton>
+          <TabButton active={activeTab === Tabs.DATASETS} onClick={() => {
+            this.setState({
+              activeTab: Tabs.DATASETS
+            });
+          }}>Datasets</TabButton>
+          <TabButton active={activeTab === Tabs.COHORTS} onClick={() => {
+            this.setState({
+              activeTab: Tabs.COHORTS
+            });
+          }}>Cohorts</TabButton>
+          <TabButton active={activeTab === Tabs.CONCEPTSETS} onClick={() => {
+            this.setState({
+              activeTab: Tabs.CONCEPTSETS
+            });
+          }}>Concept Sets</TabButton>
+        </div>
+        <div style={{
+          borderBottom: '1px solid #525A65',
+          marginLeft: '-1rem',
+          marginRight: '-1rem',
+          opacity: 0.24
+        }}>
+        </div>
+        <div style={{
+          marginBottom: '1rem',
+          display: 'flex',
+          flexWrap: 'wrap',
+          position: 'relative',
+          minHeight: 247,
+          padding: '0 0.5rem'
+        }}>
+          {filteredList.map(resource => {
+            {resourceKey = resourceKey + 1; }
```

Comment:
So I think you can just add an index to this map and use that instead of using this other variable (and adding one to it). Could be wrong...